win/win32service.cpp

```diff
@@ -196,7 +196,7 @@ void InstallService()
 		CloseServiceHandle(SCMHandle);
 		std::cout << "Service installed." << std::endl;
 	}
-	catch(CWin32Exception e)
+	catch(CWin32Exception &e)
```

Comment:
```suggestion
	catch(const CWin32Exception& e)
```

... would be more in line with our coding style.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4